src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -111,5 +111,11 @@
       <Name>VoidMainWithExitCodeApp</Name>
     </ProjectReference>
   </ItemGroup>
+  <!-- ToDo: Remove this target once the infrastructure for testing in netcoreapp1.1 is ready -->
+  <Target Name="RemoveUnwantedTestMonikers" Condition="'$(TargetGroup)'=='netstandard1.7'" BeforeTargets="CopyTestToTestDirectory">
```

Comment:
@joperezr I'm starting to get a little worried about the number of hacks we are introducing to get these tests running. Can we at a minimum put them in the root dir.targets so we don't have a bunch of these floating in all our test projects?